tests/sqllogictests/suites/base/06_show/06_0005_show_functions

```diff
@@ -11,4 +11,12 @@ to_day_of_month 1 0 (empty) (empty)
 to_day_of_week 1 0 (empty) (empty)
 to_day_of_year 1 0 (empty) (empty)
 
+query TBBTT
+SHOW FUNCTIONS LIKE 'to_day%' LIMIT 1
+----
+to_day_of_month 1 0 (empty) (empty)
 
+query TBBTT
+SHOW FUNCTIONS WHERE name='to_day_of_year' LIMIT 1
```

Comment:
Better to add a error test:

e.g. this query need to failed.

show functions where c='a';